reana_workflow_controller/rest/workflows_status.py

```diff
@@ -522,7 +522,16 @@ def set_workflow_status(workflow_id_or_name):  # noqa
             )
         elif status == DELETED:
             all_runs = True if request.json.get("all_runs") else False
-            workspace = True if request.json.get("workspace") else False
+            workspace = True if request.json.get("workspace", True) else False
+            if not workspace:
+                return (
+                    jsonify(
+                        {
+                            "message": "Workspace must be deleted with the workflow. Use workspace=True.",
```

Comment:
```suggestion
                            "message": "Workspace must always be deleted when deleting a workflow.",
```
Given that this message will be shown to users having an old version of `reana-client`, what do you think about changing it to remove any reference to the `workspace` parameter?